libs/core/langchain_core/output_parsers/pydantic_v2.py

```diff
@@ -0,0 +1,55 @@
+import json
```

Comment:
A bit worried about getting this into the code base. Definitely into `core` as it will create duplicated code in `core`  when it upgrades to pydantic 2

Not sure, but this could potentially create issues when using in a chain or a runnable map together since it'll cause mixing of pydantic v1 and v2 code.